### PR TITLE
Updated azure-mgmt-monitor to 0.7.0 and added new action-group features

### PR DESCRIPTION
**Pull request by ritwik8119**
-Updated azure-monitor-mgmt to 0.7.0
-Added support for following receivers: ArmRoleReceiver, AutomationRunbookReceiver, AzureAppPushReceiver, AzureFunctionReceiver, ItsmReceiver, LogicAppReceiver, VoiceReceiver

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
@MyronFanQiu  please help with this PR.

File: .gitignore [hunk 1]
```diff
@@ -114,3 +114,5 @@ pip.log
 
 az_command_coverage.txt
 
+/azure-cli1.pyproj
```
Comment (MyronFanQiu) [inline .gitignore]: could we remove this file?

File: src/azure-cli/azure/cli/command_modules/monitor/operations/action_groups.py [hunk 2]
```diff
@@ -26,15 +26,38 @@ def filter_receivers(collection):
...
-        from azure.mgmt.monitor.models import EmailReceiver, SmsReceiver, WebhookReceiver
+        from azure.mgmt.monitor.models import EmailReceiver, SmsReceiver, WebhookReceiver, \
+            ArmRoleReceiver, AzureAppPushReceiver, ItsmReceiver, AutomationRunbookReceiver, \
+            VoiceReceiver, LogicAppReceiver, AzureFunctionReceiver
         for r in add_receivers:
             if isinstance(r, EmailReceiver):
                 instance.email_receivers.append(r)
             elif isinstance(r, SmsReceiver):
                 instance.sms_receivers.append(r)
             elif isinstance(r, WebhookReceiver):
```
Comment (MyronFanQiu) [inline src/azure-cli/azure/cli/command_modules/monitor/operations/action_groups.py]: if or elif?

Comment (ritwik8119) [reply src/azure-cli/azure/cli/command_modules/monitor/operations/action_groups.py]: Updated to elif.

File: .gitignore (hunk 1 above)
Comment (ritwik8119) [reply .gitignore]: Removed.

File: src/azure-cli/azure/cli/command_modules/monitor/_help.py [hunk 3]
```diff
@@ -25,13 +25,42 @@
...
+        Azure Function:
+            Format:     --action azurefunction NAME FUNCTION_APP_RESOURCE_ID FUNCTION_NAME HTTP_TRIGGER_URL [usecommonalertschema]
+            Example:    --action azurefunction test_function test_rsrc test_func http://trigger usecommonalertschema
+        Multiple actions can be specified by using more than one `--add-action` argument.
+        'useaadauth', 'isglobalrunbook' and 'usecommonalertschema' are optional arguements that only need to be passed to set the respective parameter to True.
+        If the 'useaadauth' argument is passed, then the OBJECT_ID and IDENTIFIER_URI values are required as well.
   - name: --short-name
     short-summary: The short name of the action group
-examples:
+example:
```
Comment (MyronFanQiu) [inline src/azure-cli/azure/cli/command_modules/monitor/_help.py]: Hello. Is this intentional? I think it should be `examples`

Comment (ritwik8119) [reply src/azure-cli/azure/cli/command_modules/monitor/_help.py]: Corrected the typo.

**Comment by MyronFanQiu** [comment]
Thanks for your contribution!

**Pull request merged**